capture/src/capture.cpp

```diff
@@ -81,6 +82,16 @@ int main(int argc, char *argv[])
 
     printf("main thread id: %ld\n", syscall(SYS_gettid));
 
+    const char *cam_name = nullptr;
+    const char *filename = nullptr;
+    for (int i = 1; i < argc; ++i) {
+        if (strncmp(argv[i], "camera=", 7) == 0) {
+            cam_name = argv[i] + 7;
+        } else if (strncmp(argv[i], "file=", 5) == 0) {
+            filename = argv[i] + 5;
+        }
+    }
```

Comment:
I suppose I could live with this. I've used and been reasonably happy with `boost::program_options`. For this simple situation it shouldn't be too bad to integrate.

Reply:
Yeah I think I should go ahead and reimplement this with `boost::program_options`.